CMakeLists.txt

```diff
@@ -131,13 +137,6 @@ if(CMAKE_VERSION VERSION_LESS "3.7.0")
   set(CMAKE_INCLUDE_CURRENT_DIR ON)
 endif()
 
-if(CMAKE_CXX_COMPILER_ID MATCHES "[Cc]lang")
-  # using regular Clang or AppleClang
-  set(CLANG ON)
```

Comment:
There are still some places using CLANG.
Does this  still work? I did not find it in the cmake docs after a brief look.

Reply:
Oh, I forgot to search for `if(CLANG`. I will drop the commit.